ui/cypress/e2e/variables.test.ts

```diff
@@ -139,9 +138,67 @@ describe('Variables', () => {
 
     cy.getByTestID('rename-variable-submit').click()
 
-    cy.get('.cf-resource-name--text').should($s =>
-      expect($s).to.contain('-renamed')
+    cy.getByTestID('notification-success--dismiss').click()
+
+    cy.getByTestID(`variable-card--name Little Variable-renamed`).should(
+      'exist'
     )
+
+    // Create a Map variable from scratch
+    cy.getByTestID('add-resource-dropdown--button').click()
+
+    cy.getByTestID('add-resource-dropdown--new').click()
+
+    cy.getByTestID('variable-type-dropdown--button').click()
+    cy.getByTestID('variable-type-dropdown-map').click()
```

Comment:
i've had trouble in the past with dropdown clicking. no real feedback or advice, just a reminder that clicking dropdown menus seems to be a source of flake :+1: